pydy/tests/test_system.py

```diff
@@ -7,16 +7,18 @@
 import sympy as sm
 from sympy.physics.mechanics import dynamicsymbols
 from scipy.integrate import odeint
-theano = sm.external.import_module('theano')
 
 from ..system import System
 from ..models import multi_mass_spring_damper, n_link_pendulum_on_cart
 from ..utils import sympy_equal_to_or_newer_than, PyDyImportWarning
 
+theano = sm.external.import_module('theano')
```

Comment:
I see. Well, this is a nonstandard import because we use the custom function. PEP8 are guidelines, not strict rules. In this case I think you should leave it as it was.